src/v2/Apps/Conversation/Components/__tests__/Item.jest.tsx

```diff
@@ -1,10 +1,10 @@
-import { mount } from "enzyme"
+import { screen, render } from "@testing-library/react"
 import { Item } from "../Item"
 import { Conversation_conversation } from "v2/__generated__/Conversation_conversation.graphql"
 
 describe("Item", () => {
   describe("when inquiry item is an artwork", () => {
-    // @ts-expect-error STRICT_NULL_CHECK
+    // @ts-ignore
```

Comment:
I'm still not 100% which to use, I saw other reviews people asking to use @ts-ignore instead of the first.

Reply:
`// @ts-ignore` is correct! Here's the backstory: https://github.com/artsy/force/blob/master/docs/best_practices.md#avoid-copying-and-try-to-fix--ts-expect-error-strict_null_check-flags

Reply:
One thing however, is rather than switch to `// @ts-ignore` we'll want to try to fix the error. With the `STRICT_NULL_CHECK` flag we know that this was added by the codemod, and is waiting for a user to address.